datahub-frontend/app/react/resolver/GetChartsResolver.java

```diff
@@ -34,24 +34,31 @@ public GetChartsResolver(final AnalyticsService analyticsService) {
 
   @Override
   public final List<AnalyticsChartGroup> get(DataFetchingEnvironment environment) throws Exception {
```

Comment:
This may clutter the UI. Can we make a change where we either have a drop down to toggle between the two views or have two tabs? @gabe-lyons